add Historical Properties text (#140)
intarchboard/program-rfced-future: draft-iab-rfcefdp-rfced-model.md

The third paragraph of the "Status of This Memo" section shall be
as specified in {{Section 3.5 of RFC7841}}.

# Historical Properties of the RFC Series {#properties}

This section lists some of the properties that have been 
historically regarded as important to the RFC Series. Proposals 
that affect these properties are possible within the processes 
defined in this document. As described under {{workflow}} and {{cfc}},

> proposals that might have a detrimental affect on these properties

Author: ```suggestion
proposals that might have a detrimental effect on these properties
```

should receive heightened scrutiny during RSWG discussion and RSAB 
review. The purpose of this scrutinty is to ensure that all changes are 
deliberate and that the consequences of a proposal, as far as they can be
identified, have been carefully considered.

## Availability

> Documents in the RFC Series have been available for more than 35 years, 
> with no fee for access.

Reviewer: JCK mentions that the series is >50yo.  This is true, but it's not necessary to be so precise:

```suggestion
Documents in the RFC Series have been available for many decades, 
with no fee for access.
```

## Accessibility

RFC Series documents have been published in a format that was intended 
to be as accessible as possible to those with special needs, e.g., for 
those with impaired sight.

## Language

All existing RFC Series documents have been published in English. 

> Documents have been licensed to explicitly permit translation into 
> other languages.

Reviewer: A lightly edited version of @adamroach's suggestion:

```suggestion
Documents have been made available under terms that allow anyone
to translate them into other languages without asking for permission.
```

## Diversity

> In addition to Internet standards, the RFC series has published procedural 
> and informational documents, thought experiments, speculative ideas, 
> research papers, histories, humor, and even eulogies.

Reviewer: JCK suggests (and I think this is better):

```suggestion
The RFC series has included many types of documents including standards for
the Internet, procedural and informational documents, thought experiments,
speculative ideas, research papers, histories, humor, and even eulogies.
```

## Quality

RFC Series documents have been reviewed for subject matter quality and 
edited by professionals with a goal of ensuring that documents are clear, 
consistent, and readable {{RFC7322}}.

## Stability

Once published, RFC Series documents have not changed.

## Longevity

RFC Series documents have been published in a form intended to be easily 

> legible to humans for decades or longer.

Reviewer: JCK's suggestion:

```suggestion
accessible and usable to humans for decades or longer.
```

The third paragraph of the "Status of This Memo" section shall be
as specified in {{Section 3.5 of RFC7841}}.

# Historical Properties of the RFC Series {#properties}

This section lists some of the properties that have been 
historically regarded as important to the RFC Series. Proposals 
that affect these properties are possible within the processes 
defined in this document. As described under {{workflow}} and {{cfc}},
proposals that might have a detrimental effect on these properties
should receive heightened scrutiny during RSWG discussion and RSAB 
review. The purpose of this scrutiny is to ensure that all changes are 
deliberate and that the consequences of a proposal, as far as they can be
identified, have been carefully considered.

## Availability

Documents in the RFC Series have been available for many decades, 

> with no fee for access.

Reviewer: ```suggestion
with no constraints on access or distribution.
```

Reviewer: or
```suggestion
with no constraints on access or redistribution.
```

## Accessibility

RFC Series documents have been published in a format that was intended 
to be as accessible as possible to those with special needs, e.g., for 
those with impaired sight.

## Language

All existing RFC Series documents have been published in English. 
However, since the beginning of the RFC series, translation into a 

> language other than English has been allowed and encouraged. The IETF 
> Trust and the RFC Editor place no restrictions on this. 

Reviewer: ```suggestion
language other than English has been allowed and encouraged.
```

## Diversity

The RFC series has included many types of documents including standards for
the Internet, procedural and informational documents, thought experiments,
speculative ideas, research papers, histories, humor, and even eulogies.

## Quality

RFC Series documents have been reviewed for subject matter quality and 
edited by professionals with a goal of ensuring that documents are clear, 
consistent, and readable {{RFC7322}}.

## Stability

Once published, RFC Series documents have not changed.

## Longevity

RFC Series documents have been published in a form intended to be easily 

> accessible and usable to humans for decades or longer.

Reviewer: Based on discussion on-list, a thought:

```suggestion
comprehensible to humans for decades or longer.
```

Reviewer: Stephen points out that pairing that word with "easily" might not be wise.  For some reason I can't re-suggest here (GitHub failure), so maybe consider that also.
